site/versions/master/docs/install.md

```diff
@@ -136,25 +136,29 @@ You can also add this command to your `~/.bashrc` file.
 
 Install Bazel on Mac OS X using one of the following methods:
 
-  * [Using Homebrew](#install-on-mac-os-x-homebrew)
-  * [Using binary installer](#install-with-installer-mac-os-x)
+  * [Install using Homebrew](#install-on-mac-os-x-homebrew)
+  * [Install with installer](#install-with-installer-mac-os-x)
   * [Compiling Bazel from source](#compiling-from-source)
 
 
-### <a name="install-on-mac-os-x-homebrew"></a>Using Homebrew
+### <a name="install-on-mac-os-x-homebrew"></a>Install using Homebrew
 
-#### 1. Install Homebrew on Mac OS X (one time setup)
+#### 1. Install JDK 8
 
-`$ /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`
+JDK 8 can be downloaded from
+[Oracle's JDK Page](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html).
+Look for "Mac OS X x64" under "Java SE Development Kit". This will download a
+DMG image with an install wizard.
 
-#### 2. Install Bazel Homebrew Package
+#### 2. Install Homebrew on Mac OS X (one time setup)
 
-`$ brew install bazel`
+`$ /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`
 
-Once installed, you can upgrade to newer version of Bazel with:
+#### 3. Install Bazel Homebrew Package
 
-`$ brew upgrade bazel`
+`$ brew install bazel`
 
+You are all set. You can confirm Bazel is install successfully by running `$ bazel`.
```

Comment:
s/install/installed/

Also, `bazel version` instead of `$ bazel` (don't include the $ for inline snippets).